### DISCUSSION TITLE
Nuxt 3 Roadmap ?

### DISCUSSION THREAD

**Discussion by jforatier** [nuxt-community/firebase-module#598] [Ideas]
Hi,
Is there any roadmap, branch, issue or brainstorm around [Nuxt 3](https://v3.nuxtjs.org/) support ?
Maybe there is a two-steps plan :
- First target [Bridge](https://v3.nuxtjs.org/getting-started/bridge)
- then target Full Nuxt/Vue v3

Thanks for the great job 🥇

**Comment by lupas** [comment]
@jforatier 
Currently nothing really planned. Would probably tackle it in the same rewrite of the module that is needed for switching to supporting the modular syntax of Firebase 9+ (see https://github.com/nuxt-community/firebase-module/issues/597), but there is currently no plan/roadmap for that.

If you or anyone finds time to get this thing started I am more than supportive of that.

Let's see, I might find some time before the end of the year for a first approach :)

**Comment by JenuelDev** [comment]
How is the support for nuxt 3?

**Comment by kai-oswald** [comment]
Hello everyone, I have worked on a Nuxt3 - module for the past few weeks and it's currently in a usable state, but I will probably need some help for a few issues that I don't know how to solve.
Any feedback is greatly appreciated.

- Nuxt 3 compatible
- Modular mode (firebase v9)
- Vue composables

https://github.com/kai-oswald/nuxt-firebase
